selenium_tests/test_storage.py

```diff
@@ -0,0 +1,86 @@
+import json
+import pathlib
+import os
+import time
+
+from selenium import webdriver
+from django.test import LiveServerTestCase, tag
+from selenium.webdriver.chrome.options import Options
+import clipboard
+
+URL = "http://localhost:8000/"
+
+
+@tag('storage')
+class TestStorage(LiveServerTestCase):
+    def __init__(self, methods):
+        super().__init__(methods)
+        self.chrome_options = Options()
+        self.chrome_options.add_argument("--headless")
+        self.driver = webdriver.Chrome(chrome_options=self.chrome_options)
+
+    def setUp(self):
+        self.driver.get(URL)
+
+    def tearDown(self):
+        self.driver.quit()
```

Comment:
- Inherit from the SeleniumTestBase
- remove last three lines of constructor
- remove setup, teardown methods